StateObserver/StateObserver.cpp

```diff
@@ -589,6 +591,7 @@ namespace WPEFramework {
 				}
 
 			}
+		}
 
 			response["properties"]=response_arr;
```

Comment:
In case of error, what would be the output of the json request. Make sure the response still comes as valid error message.